client/lib/feed-stream-store/test/index.js

```diff
@@ -1,3 +1,11 @@
+jest.mock( 'lib/analytics', () => ( {} ) );
+jest.mock( 'lib/data-poller', () => require( './lib/data-poller' ) );
+jest.mock( 'lib/post-normalizer', () => require( './lib/post-normalizer' ) );
+jest.mock( 'lib/wp', () => require( './lib/wp' ) );
+jest.mock( 'reader/stats', () => ( {
```

Comment:
@nb they're mostly Reader-specific wrappers around `mc, ga, tracks` from `lib/analytics`, so thought they might be best off inside `client/reader`. Would be happy to move to `lib/analytics/reader.js` perhaps, if that makes more sense?